content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,26 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/aws-load-balancer-security-groups (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f, sg-8725gr62r"`
+
+Used on: Service
+
+The cloud controller manager integration with AWS elastic load balancing configures
+this annotation to specify a comma seperated list of frontend securityGroups you want
+to attach to an NLB. Both name and ID of securityGroups are supported where name matches
+a `Name` tag, not the `groupName` attribute.
+
+When not present, the controller will automatically create one security group.
```

Comment:
"When not present", what is not present?
This is not a complete sentence.

Reply:
This is meant to indicate the condition when the annotation and its value are not present.